crates/chronicle/Cargo.toml

```diff
@@ -7,47 +7,48 @@ version = "0.7.5"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-Inflector            = { workspace = true }
-api                  = { path = "../api" }
-async-graphql        = { workspace = true }
-async-stl-client     = { workspace = true }
-cfg-if               = { workspace = true }
-chronicle-protocol   = { path = "../chronicle-protocol" }
-chronicle-telemetry  = { path = "../chronicle-telemetry" }
-chrono               = { workspace = true, features = ["serde"] }
-clap                 = { workspace = true, features = ["derive", "env"] }
-clap_complete        = { workspace = true }
-colored_json         = { workspace = true }
-common               = { path = "../common" }
-const_format         = { workspace = true }
-diesel               = { workspace = true }
-dotenvy              = { workspace = true }
-futures              = { workspace = true }
-genco                = { workspace = true }
-hex                  = { workspace = true }
-iref                 = { workspace = true }
-is-terminal          = { workspace = true }
-jsonschema           = { workspace = true }
-opa                  = { workspace = true }
-opentelemetry        = { workspace = true }
-percent-encoding     = { workspace = true }
-question             = { workspace = true }
-rand                 = { workspace = true }
-rand_core            = { workspace = true }
-serde                = { workspace = true }
-serde_derive         = { workspace = true }
-serde_json           = { workspace = true }
-serde_yaml           = { workspace = true }
-shellexpand          = { workspace = true }
-thiserror            = { workspace = true }
-tokio                = { workspace = true }
-toml                 = { workspace = true }
-tracing              = { workspace = true }
-tracing-log          = { workspace = true }
-url                  = { workspace = true, features = ["serde"] }
-user-error           = { workspace = true }
-uuid                 = { workspace = true }
-valico               = { workspace = true }
+Inflector           = { workspace = true }
```

Comment:
Shall we just start using default formatting so it's easier to see the actual changes?

Reply:
Lets try to align it - I am using an extension currently - Even Better Toml,
there may be a CLI equivalent. It's annoying rustfmt doesn't

Reply:
CLI would be great but, for the meantime, we too could try the extension.